### PR TITLE
Solarboards

### PR DESCRIPTION
**Pull request by ValkyrieSkies**
Tracker Electronics are currently only available via cargo's solar panel supply pack, so I've added them to the imprinter's engineering boards.

Also minor grammar tweak just because I spotted it.

Tested, works fine.

:cl:
 * rscadd: Added Tracker Electronics to the Circuit Imprinter

### CODE REVIEW

**Comment by SonixApache** [comment]
something something soldering iron

**Comment by ValkyrieSkies** [comment]
Solar Control Console is already on the Imprinter so separating them doesn't make much sense.

**Comment by ghost** [comment]
I think what @SonixApache means is that soldering iron has many such minor engineering circuits and all soldering iron circuits are also on circuit imprinter, so it would make sense to have the tracker on both.

You can set solars to mimic autotrack by using very basic number division and setting correct manual tracking values, if you're worried about tracker circuit being too complicated for soldering iron.

**Comment by ValkyrieSkies** [comment]
Sucessfully tested, works just fine.

File: code/modules/research/designs/boards/machine_engie.dm [hunk 1]
```diff
@@ -46,14 +46,24 @@
 
 /datum/design/cell_charger
 	name = "Circuit Design (Cell Charger)"
-	desc = "Allows for the construction of circuit boards used to build a cell charger"
+	desc = "Allows for the construction of circuit boards used to build a Cell Charger."
```
Comment (jknpj) [inline code/modules/research/designs/boards/machine_engie.dm:49]: why

Comment (ValkyrieSkies) [reply code/modules/research/designs/boards/machine_engie.dm:49]: Unatomic but it's just to bring that bit of text more in line with other instances of that style.

**Comment by ValkyrieSkies** [comment]
Merge me coward.

**Comment by ValkyrieSkies** [comment]
It has been a week.

**Pull request merged**